pages/integrations/amazon_eventbridge.md

```diff
@@ -692,3 +693,45 @@ AWS EventBridge has strict limits on the size of the payload as documented in [A
   }
 }
 ```
+
+<h3 id="audit-event-logged">Audit Event Logged</h3>
+
+<%= pill "COMING SOON", "march-23-release" %> [Audit log](/docs/pipelines/audit-log) is only available to Buildkite customers on the [Enterprise](https://buildkite.com/pricing) plan
```

Comment:
Similarly something like this:

![Screenshot 2023-03-13 at 19 35 30](https://user-images.githubusercontent.com/395/224648460-8608fa46-e7bb-4d6d-bcab-5a3f2d38818f.png)